File: revition/traversal-pattern/isSorted.java
public class Main{
    public static void main(String[] args){
        boolean isSorted = true;
        int[] arr = {1,2,3,4,5};
        int [] arr2 = {5,4,2,1,9};
        for(int i = 0; i<arr2.length -1; i++){
            if(arr2[i] > arr2[i+1]){
                isSorted = false;
            }
        }
        if(isSorted){
            System.out.println("The Array is Sorted");
        }
        else{
            System.out.println("The Array is Not Sorted");
        }
    } 
}

File: array/level-1/isSorted.js
function isSorted(arr){
    for(let i = 0; i<arr.length; i++){
        if(arr[i] > arr[i+1]){
            return false;
        }
    }
    return true;
}
console.log(isSorted([2,1,5,3,2]));
console.log(isSorted([1,2,3,4,5]));
/*
10. Check if an Array is Sorted
Problem

Given an array, determine whether the elements are arranged in ascending order.

Example 1
Input:
[1, 2, 3, 4, 5]

Output:
Sorted

Because:

1 < 2 < 3 < 4 < 5
Example 2
Input:
[1, 3, 2, 4, 5]

Output:
Not Sorted

Because:

3 > 2

The order is broken.
*/